ipatests/test_webui/test_service.py

```diff
@@ -218,7 +218,7 @@ def test_certificates(self):
         self.wait()
         self.select('select', '6')
         self.dialog_button_click('ok')
-        self.wait_for_request(n=2, d=3)
+        self.wait_for_request(1, n=3, d=3)
```

Comment:
Is `wait_for_request` the right call(wait) here? It checks for `.global-activity-indicator` wouldn't it be better to check for the spinner (`.working-widget`) in the cert_widget - as we can see that it is still spinning in the [screenshot](http://freeipa-org-pr-ci.s3-website.eu-central-1.amazonaws.com/jobs/c78d72fa-15b0-11e9-8714-fa163ea17f49/2019-01-11T15:24:40.110578_test_service_test_certificates.png)

Reply:
Your right. I'll change this wait function.

Reply:
@pvoborni
please check if the new method is ok